config/routes.rb

```diff
@@ -1,5 +1,11 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  resources :flights do
+    delete 'remove_passenger/:id', to: 'flights#destroy', as: 'remove_passenger'
+  end
```

Comment:
Your delete route is incorrect and unRESTful. The verb, and action are correct. The path and controller are not. Your path should follow nested resource routing conventions and the request should not be going to the flights controller since we are not deleting a row in the flights table.